x-pack/legacy/plugins/siem/public/containers/case/api.ts

```diff
@@ -130,3 +130,25 @@ export const patchComment = async (
   await throwIfNotOk(response.response);
   return convertToCamelCase<CommentResponse, Comment>(decodeCommentResponse(response.body));
 };
+
+export const deleteCases = async (caseIds: string[]): Promise<boolean> => {
+  const response = await KibanaServices.get().http.fetch<string>(`${CASES_URL}`, {
+    method: 'DELETE',
+    asResponse: true,
+    query: { ids: JSON.stringify(caseIds) },
```

Comment:
I think we do not need to stringify here. It is taken care of fetch.

Reply:
if I do not stringify I get this error:
```
{
  "statusCode":400,
  "error":"Bad Request",
  "message":"[request query.ids]: could not parse array value from json input"
}
```

Reply:
Ok! Sorry, my mistake.